packages/vue/src/components/molecules/SfBanner/SfBanner.vue

```diff
@@ -1,10 +1,30 @@
 <template>
-  <section
-    class="sf-banner"
-    :style="style"
-    v-on="isMobileView ? $listeners : {}"
-  >
+  <section class="sf-banner" :style="style" v-on="$listeners">
     <component :is="wrapper" class="sf-banner__wrapper" :link="link">
```

Comment:
Without mobile observer we can have div here - it won't change on button or link

Reply:
@justyna-13 We have div for desktop, but for mobile it must be a link/button as the whole banner is clickable then.

Reply:
right :) changed condition looks good